### PR TITLE
fix: ai translations description for atoms

### PR DESCRIPTION
**Pull request by Ryukemeister**
## What does this PR do?

- Translations for event types description using ai was causing atoms to break, hence removed ai translations for atoms for the time being.

## Mandatory Tasks (DO NOT REMOVE)

- [ ] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] N/A - I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

build platform atoms

### CODE REVIEW

File: packages/features/bookings/Booker/Booker.tsx [hunk 1]
```diff
@@ -358,6 +355,7 @@ const BookerComponent = ({
                   event={event.data}
                   isPending={event.isPending}
                   isPlatform={isPlatform}
+                  i18nLocales={i18nLocales}
```
Comment (hbjORbj) [inline packages/features/bookings/Booker/Booker.tsx:358]: double checking: you don't need to pass `userLocale` here?

File: packages/features/bookings/Booker/components/EventMeta.tsx [hunk 2]
```diff
@@ -61,8 +61,9 @@ export const EventMeta = ({
     eventMetaTitle?: string;
     eventMetaTimezoneSelect?: string;
   };
+  locale?: string | null;
```
Comment (hbjORbj) [inline packages/features/bookings/Booker/components/EventMeta.tsx:64]: shouldn't this be `userLocale`? based on the other changes you added

File: packages/platform/atoms/booker/BookerWebWrapper.tsx [hunk 3]
```diff
@@ -219,6 +225,9 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
       schedule={schedule}
       verifyCode={verifyCode}
       isPlatform={false}
+      areInstantMeetingParametersSet={areInstantMeetingParametersSet}
+      userLocale={session?.user.locale}
+      i18nLocales={i18nLocales}
```
Comment (hbjORbj) [inline packages/platform/atoms/booker/BookerWebWrapper.tsx:230]: Hmm passing a prop doesn't seem optimal. I'd suggest importing it from `i18n.json` where it's needed

**Review by hbjORbj — DISMISSED** [review]

File: packages/features/bookings/Booker/components/EventMeta.tsx [hunk 4]
```diff
@@ -107,11 +107,11 @@ export const EventMeta = ({
     : isHalfFull
     ? "text-yellow-500"
     : "text-bookinghighlight";
-  const userLocale = session.data?.user.locale ?? navigator.language;
+  const userLocale = locale ?? navigator.language;
   const translatedDescription = (event?.fieldTranslations ?? []).find(
     (trans) =>
       trans.field === EventTypeAutoTranslatedField.DESCRIPTION &&
-      i18nLocales.includes(trans.targetLocale) &&
+      i18nLocales.locale.targets.includes(trans.targetLocale) &&
```
Comment (hbjORbj) [inline packages/features/bookings/Booker/components/EventMeta.tsx:114]: 1. i18nLocales name is misleading
2. `i18n.locale.targets.concat([i18n.locale.source])`

**Review by ThyMinimalDev — APPROVED** [review]

**Pull request merged**